test/sources/solidity/contracts/comments/postFunctionDeclarationComment.sol

```diff
@@ -3,5 +3,7 @@ pragma solidity ^0.5.0;
 contract Test {
   function a(bool test) public {//Comment immediately after function declaration
   }
-  function b(bool test) public {uint8 x=1;}//Comment immediately after function closes
```

Comment:
We need an additional test case here - `{uint8 x=1;}` directly after the fn def trigger an instrumentation error.